### PR TITLE
Update for sage release notes

### PR DESCRIPTION
**Pull request by j-wags**
Sage image license should be appropriate - From https://pixabay.com/photos/sage-kraeuer-seasoning-sage-leaves-74326/. I'm not tied to it, so if anyone finds something better feel free to replace it. 

I could use a proofread on this -- @SimonBoothroyd, could you take a look? Feel free to replace it entirely if you have better text from somewhere else.

### CODE REVIEW

File: content/force-fields/force-fields/_index.md [hunk 1]
```diff
@@ -9,14 +9,34 @@ class: small-images
...
 
+All currently available versions are listed below. Each force field is currently available in two forms - with and without bond constraints to hydrogen. The default version of each force field is suitable for typical molecular dynamics simulations with constrained bonds to hydrogen. The unconstrained version of each force field should be used when single-point energies are a major concern (e.g. geometry optimizations) and when comparing the force field to QM data.
+
+{{< br >}}
+## Sage ##
+![Sage](img/sage.jpg)
+### Versioning: openff-2.y.z ###
+
+_Sage_ (OpenFF-2.y.z) is the codename for the second generation of optimized small molecule force fields from the Open Force Field Initiative. Sage contains a refit of the Lennard-Jones parameters from the AMBER-lineage _smirnoff99Frosst_ to physical property datasets, in addition to a refit of valence parameters from Parsley 1.3.0.
```
Comment (SimonBoothroyd) [inline content/force-fields/force-fields/_index.md:19]:
```suggestion
_Sage_ (OpenFF-2.y.z) is the codename for the second generation of small molecule force fields from the Open Force Field Initiative. Building upon Parsley 1.3.0, it contains a partial re-training of the Lennard-Jones parameters against experimental physical property data in addition to a set of retrained valence parameters .
```

File: content/force-fields/force-fields/_index.md [hunk 2]
```diff
@@ -9,14 +9,34 @@ class: small-images
...
+
+{{< br >}}{{< br >}}
+### openff-2.0.0 ###
+
+#### Release date: August 16, 2021
+
+This is the first release of the Sage line of force fields. Full details of the Sage fitting are available in the releases section of the [openff-sage GitHub repository](https://github.com/openforcefield/openff-sage/releases/)
```
Comment (SimonBoothroyd) [inline content/force-fields/force-fields/_index.md:26]:
```suggestion
This is the first release of the Sage line of force fields. Full details of the Sage training and assessment pipelines are available in the releases section of the [openff-sage GitHub repository](https://github.com/openforcefield/openff-sage/releases/)
```

**Review by SimonBoothroyd — APPROVED** [review]
A few minor suggestions but otherwise LGTM!

**Comment by karmencj** [comment]
Ah, darn, I didn't see this pull request -- I've just added an entry for Sage, but please feel free to change it. See [here](https://openforcefield.org/force-fields/force-fields/) @j-wags @SimonBoothroyd

**Comment by j-wags** [comment]
Forgot why this is still open, but since it's more than 6 months old I assume it's not that important.

**Pull request closed without merging**